edx_rest_framework_extensions/auth/jwt/middleware.py

```diff
@@ -112,17 +114,21 @@ def process_request(self, request):
         """
         Reconstitute the full JWT and add a new cookie on the request object.
         """
+        use_jwt_cookie = request.META.get(USE_JWT_COOKIE_HEADER)
         header_payload_cookie = request.COOKIES.get(jwt_cookie_header_payload_name())
         signature_cookie = request.COOKIES.get(jwt_cookie_signature_name())
 
-        # Reconstitute JWT auth cookie if split cookies are available.
-        if header_payload_cookie and signature_cookie:
+        if not use_jwt_cookie:
+            metric_value = 'not-requested'
```

Comment:
Nit: rename the `use_jwt_cookie` to `use_jwt_cookie_requested` - it will then be consistent with the metric value and may be clearer.